{% include "header.html" %}

<div id="content" style="text-align:justify">
    {% include "masthead.html" %}
    <section id="main-content-area" class="container" role="main">
        <nav aria-label="You are here:" role="navigation">
            <ul class="breadcrumbs columns">
                <li>FunPDBe</li>
                <li>
                    <span class="show-for-sr">Current: </span> About FunPDBe
                </li>
            </ul>
        </nav>

        <div class="container">
            <h2>FunPDBe</h2>
            <h3>Community driven enrichment of PDB data with structural and functional annotations</h3>

            <div id="summary">
                <p>The FunPDBe project aims to establish an integrated and easily accessible resource of structural
                    and functional annotations for macromolecular structure data in the Protein Data Bank (PDB).
                    It is a collaboration between the Protein Data Bank in Europe (PDBe) and world-leading UK-based
                    providers of structural bioinformatics data. The project promotes interoperability, comparative
                    analysis and exchange of structural and functional annotations by implementing common data
                    standards and infrastructure to collect these enhanced annotations. The project aims to
                    significantly increase the impact of structural data globally by implementing a central sustainable
                    data resource and a uniform data access mechanism (via FTP and REST API) for distribution of these
                    valuable functional and structural annotations, developed by researchers in the UK. The data will
                    also be made accessible via web interface by developing re-usable web components.</p>
            </div>

            <hr>

            <div id="partners" class="row">
                <h3>FunPDBe partner resources</h3>
                {% for group in group_info %}
                    <div class="col-sm-12 col-md-4">
                        <h4>
                            <a href="{{group.url}}" target="_blank">
                                {{group.name}}
                                <i class="icon icon-generic" data-icon="x"></i>
                            </a>
                        </h4>
                        <strong>{{group.owner}}</strong>
                        <p style="text-align: justify">{{group.description}}</p>
                    </div>
                {% endfor %}
            </div>

            <hr>

            <div id="background">
                <h4>Background</h4>
                <p>Protein Data Bank in Europe (PDBe [1]) is one of the founding members of the worldwide
                    Protein Data Bank (wwPDB) [2] and manages the single worldwide macromolecular structure archive,
                    the Protein Data Bank (PDB) [3], in collaboration with its partner sites. PDBe manages deposition
                    and annotations of PDB submissions originating from Europe and Africa. PDBe works closely with
                    its wwPDB partners to develop deposition and annotation policies and infrastructure.</p>

                <p>The model coordinates available in the PDB provide unique information including details of secondary,
                    tertiary and quaternary structure in addition to information on residue interactions, interface
                    residues and atomic level details of interactions between small molecules and a given
                    macromolecule.</p>

                <p>However, the atomic coordinates only possess biological value when they are supplemented by
                    information providing biological context and enriched by value-added annotations [4]. When combined
                    with biological knowledge and context, the coordinates can provide mechanistic information on
                    biological processes. Presently, the PDB annotation includes the following main elements:
                    information on secondary structure; basic data on quaternary structure; binding site environment;
                    cross-referencing information for each polypeptide chain to the corresponding entry from UniProt
                    and, if applicable, enzyme commission numbers; and NCBI taxonomy IDs for organism information.
                    The individual ‘small molecule’ chemical entities present in PDB entries are described in a separate
                    reference resource called the chemical component dictionary [5].</p>

                <p><strong>It is difficult to exploit to the full extent the rich data available in the PDB not only
                    because of the complexity of the data itself, but also due to the limited and static nature of
                    the annotations provided by the PDB and most importantly because the biological context, structural
                    and functional annotations pertaining to each entry are almost completely
                    lacking [6, 7].</strong> </p>

                <p>FunPDBe is designed to provide a solution that will bring together the basic annotations from the
                    PDB archive, the value-added annotations that are available at PDBe and the currently fragmented
                    structural and functional annotations from the numerous structural bioinformatics data resources
                    in the UK such as: catalytic sites (Catalytic Site Atlas – CSA [8]), enzyme reaction mechanisms
                    (Mechanisms in Enzymes – MACiE [9]), analysis of biological assemblies (3D Complex [10]),
                    predicted functional site data for ligand binding sites or interface residues (CATH- Gene3D
                    FunSites [11]; 3DLigandSite [12]; POPSCOMP [13]), nucleolar signal localisation detector
                    (NOD [14, 15]), the predictor of O-GlcNAc Sites (POGSPSF, in preparation) and the JABAWS2
                    webservices framework that integrates predictors of intrinsic disorder and conservation [16].
                    The community has also developed a number of algorithms to predict the effect of mutations or
                    variants on the structure and/or function of macromolecules [17-22].</p>
            </div>

            <div id="objectives">
                <h4>Objectives</h4>
                <p>Our objective is to provide FunPDBe, a comprehensive resource of functional annotation for structures
                    in the PDB, by combining the work and expertise of 10 leading structural computational biologists in
                    the UK. We develop the processes necessary to facilitate the provision of functional annotations
                    alongside existing PDB information by integrating value-added data from specialist data resources
                    into the PDBe infrastructure. Focus is on the following three major areas:</p>
                <ul>
                    <li>Prediction of functional sites including binding and interface sites, led by C Orengo</li>
                    <li>Annotation of ‘known’ functional sites in proteins, especially enzymes, ligand binding sites and
                        protein interfaces, led by J Thornton</li>
                    <li>Predicting the structural effects of mutations and variants, which is a critical area for
                        translational research, led by M Sternberg</li>
                </ul>
                <p>In each of the above areas a number of groups from the UK bioinformatics community participate as
                    collaborators to provide functional annotations.</p>
            </div>

            <div id="relevance">
                <h4>Why is it relevant?</h4>
                <p>From the early 1970s when the PDB was established, structural data have contributed valuable insights
                    into the function of biological macromolecules. This is evident from the number of downloads of PDB
                    entries (more than 500 million each year), and from the steady number of citations that each PDB
                    entry receives, which shows a sustained use of this information [23]. Recent analysis by wwPDB
                    partners shows that more than 25% (119 out of 425) of the databases published in the NAR database
                    issue between 2011 and 2016 use PDB information and provide valuable annotations based on structural
                    data. This analysis also points to the lack of a central repository that would provide structural
                    and functional annotation alongside the basic PDB annotation. Such fragmentation of information and
                    lack of data standards also makes it difficult to carry out large-scale data analysis.</p>

                <p>The basic data in the PDB currently include atomic coordinates, experimental data, information about
                    data collection and some basic annotation that can be easily derived from the atomic coordinates
                    (e.g., secondary structure). However, many bioinformaticians have developed valuable niche
                    resources as part of their on-going research (e.g., the ‘Catalytic Site Atlas’ that catalogues
                    enzyme catalytic residues from the literature).</p>

                <p>By bringing together functional annotations available in these different data resources in a central
                    repository, FunPDBe will support the wider research community in its efforts to exploit
                    macromolecular structure data to derive knowledge. Standardisation of the metadata and the
                    availability of multiple data access mechanisms (REST API, multiple file formats on FTP site) will
                    also support large scale data analysis by expert users, as they will not have to implement ad hoc
                    mechanisms to access each individual data resource, an effort which is often performed independently
                    by each researcher.</p>

                <p>The project will also improve sustainability, ensuring that the annotations are archived safely, are
                    accessible for the foreseeable future, remove duplication of effort and thus protect the work and
                    investment that has gone into developing these specialised data resources. FunPDBe will become a
                    unique, open global resource, and help secure the UK’s leading role in structural computational
                    biology into the future.</p>

                <p>No other resource integrates curated known functional residues with predicted functional residues
                    from diverse resources to more comprehensively identify active sites and interfaces, nor combines
                    multiple predictions on the structural and functional impacts of genetic variations. By developing
                    standards and combining predictions from multiple methods, the FunPDBe project will provide more
                    comprehensive and more reliable data than any individual resource and to a much larger set of users
                    than currently enjoyed by any of the individual resources.</p>
            </div>

            <div id="providing">
                <h4>What do we provide?</h4>
                <p>The participating bioinformatics teams have developed a number of algorithms and niche data resources
                    that provide structural and functional annotations. The annotation information available in these
                    resources is widely used as evident from the number of citations and/or by the number of users
                    accessing these data. PDBe is one of the core EMBL and Elixir data resources and has more than 550K
                    unique users visiting its website annually. The team has also developed the SIFTS data resource [24]
                    and data analysis tools (PISA [25] 3143 citations in Web of Science, PDBeFOLD [26] 2071 citations)
                    that are widely used. For instance, the latest release of Jalview (2.10, October 2016) [27] uses
                    SIFTS to obtain the residue level cross-referencing between UniProt and the PDB. Jalview has over
                    70,000 regular users worldwide and is used in teaching and research across all biological disciplines.</p>

                <p>FunPDBe provides the necessary infrastructure to bring together the basic annotation from the PDB
                    archive, annotations from SIFTS and other PDBe resources and the value-added structural and
                    functional annotations provided by the world-leading bioinformatics resources in the UK. The
                    infrastructure provides a mechanism for the participating teams to update structural and functional
                    annotation and keep the information up-to-date. It also establishes the necessary protocol for
                    other bioinformatics teams from around the globe, not initially involved in this project, to
                    contribute annotations to FunPDBe so these can be made available via the defined data standards
                    thus creating a global, open source resource for structural and functional annotation for
                    structural data. The major impact of this project is expected to come from combining the valuable
                    knowledge from the niche annotations with the archival information from the PDB and making it
                    accessible in a standardised way. By making these data available via the PDBe website and
                    standardised REST API it can reach the wider user community that accesses the PDBe website in
                    addition to the users that already access the individual data resources.</p>

                <p>FunPDBe is a unique data resource that not only brings together the most valuable annotations but
                    also helped to develop data standards to simplify the use of these data. The data standards and the
                    central data resource developed in the project should allow standardised access helping the
                    contributing teams to carry out comparative analysis and validation of predicted annotations and
                    help improve the prediction algorithms by promoting interoperability. Additionally the new
                    developments will simplify large-scale analysis of structural data and its exploitation for
                    translational research. The implementation of REST API and access via FTP should make it possible
                    for the wider user community to access these data in a straightforward fashion. To represent the
                    data on web interfaces, web components are being developed and deposited in the BioJS
                    repository [28] for wider distribution. These web components will standardise the way these data
                    are visualised and will be integrated into PDBe web pages. The standardised access to the
                    annotation information and the availability of the web components to visualise data will enable
                    all the FunPDBe participants and other major data resources such as UniProt [29], InterPro [30]
                    and Ensembl [31] to access these data and/or make them available to their users on their respective
                    websites. The project will help individual researchers gain easy access to macromolecular structure
                    data and the associated niche annotations and help ensure UK researchers stay internationally
                    competitive. The easy access to structural data and annotations will support obtaining insights
                    into the effects of genetic variations, development of new tools to aid synthetic biology,
                    enhancement in valuable annotations to enrich information available for agriculturally important
                    macromolecules and contribute to human health by aiding interpretation of gene variation.</p>
            </div>

            <div id="references">
                <h5>References</h5>
                <ol>
                    <li>Velankar, S., et al., Nucleic Acids Res, 2016. 44: p. D385-395.</li>
                    <li>Berman, H.M., et al., Nature structural biology, 2003. 10: p. 980.</li>
                    <li>Berman, H.M., et al., Nucleic Acids Res, 2000. 28: p. 235-242.</li>
                    <li>Gerstein, M., Nat Struct Biol, 2000. 7 Suppl: p. 960-963.</li>
                    <li>Westbrook, J.D., et al., Bioinformatics, 2015. 31: p. 1274-1278.</li>
                    <li>Abagyan, R., J Comput Aided Mol Des, 2012. 26: p. 9-10.</li>
                    <li>Velankar, S. and G.J. Kleywegt, Acta crystallographica D, 2011. 67: p. 324-330.</li>
                    <li>Porter, C.T., et al., Nucleic Acids Res, 2004. 32: p. D129-133.</li>
                    <li>Holliday, G.L., et al., Nucleic Acids Res, 2007. 35: p. D515-520.</li>
                    <li>Levy, E.D., et al., PLoS Comput Biol, 2006. 2: p. e155.</li>
                    <li>Das, S., et al., Bioinformatics, 2015. 31: p. 3460-3467.</li>
                    <li>Wass, M.N. and M.J. Sternberg, Proteins, 2009. 77 Suppl 9: p. 147-151.</li>
                    <li>Kleinjung, J. and F. Fraternali, Nucleic Acids Res, 2005. 33: p. W342-346.</li>
                    <li>Scott, M.S., et al., Nucleic Acids Res, 2010. 38: p. 7388-7399.</li>
                    <li>Scott, M.S., et al., BMC Bioinformatics, 2011. 12: p. 317.</li>
                    <li>Troshin, P.V., et al., Bioinformatics, 2011. 27: p. 2001-2002.</li>
                    <li>Al-Numair, N.S. and A.C. Martin, BMC Genomics, 2013. 14 Suppl 3: p. S4.</li>
                    <li>Lu, H.C., et al., Bioinformatics, 2016. 32(16): p. 2534-2536.</li>
                    <li>Pappalardo, M. and M.N. Wass, Nucleic Acids Res, 2014. 42: p. W331-336.</li>
                    <li>Pires, D.E., et al., Nucleic Acids Res, 2014. 42: p. W314-319.</li>
                    <li>Worth, C.L., et al., Nucleic Acids Res, 2011. 39: p. W215-222.</li>
                    <li>Yates, C.M., et al., J Mol Biol, 2014. 426: p. 2692-2701.</li>
                    <li>Bousfield, D., et al., F1000Res, 2016. 5(ELIXIR): 160</li>
                    <li>Velankar, S., et al., Nucleic Acids Res, 2013. 41: p. D483-489.</li>
                    <li>Krissinel, E. and K. Henrick, J Mol Biol, 2007. 372: p. 774-797.</li>
                    <li>Krissinel, E. and K. Henrick, Acta crystallographica D, 2004. 60: p. 2256-2268.</li>
                    <li>Waterhouse, A.M., et al., Bioinformatics, 2009. 25: p. 1189-1191.</li>
                    <li>Gomez, J., et al., Bioinformatics, 2013. 29: p. 1103-1104.</li>
                    <li>UniProt Consortium, Nucleic Acids Res, 2015. 43: p. D204-212.</li>
                    <li>Mitchell, A., et al., Nucleic Acids Res, 2015. 43: p. D213-221.</li>
                    <li>Yates, A., et al., Nucleic Acids Res, 2016. 44: p. D710-716.</li>
                </ol>
            </div>
        </div>
    </section>
</div>

{% include "footer.html" %}